### ISSUE TITLE
yearrange in jquery.bundle doesn't work

### ISSUE THREAD

**Issue by sheraff0** [RobinHerbots/Inputmask#1966]
Hi Robin!
Great plugin, all works just fine, but I cannot activate the **yearrange** property. Seems to be not supported in recent version...
Using _jquery.inputmask.bundle.min.js_

What am I doing wrong, or are there any alternative ways to limit year range?

```
...
	Inputmask.extendAliases({
		'dategood': {
			alias: 'datetime',
			inputFormat: "dd.mm.yyyy г.",
			placeholder: "_",
			yearrange: {
				minyear: "1900",
				maxyear: "2000"
			}
		}
	});

	$("[name='birth-date']").inputmask({
		alias: 'dategood'
	});
```

Other **inputFormat** doesn't make difference.

**Comment by RobinHerbots** [comment]
@sheraff0 , 

The yearrange is replaced by min & max options

**Comment by sheraff0** [comment]
_min & max_ works, except for... 
the _max_ option doesn't let input maximum year correctly, if the date (day & month) is greater than its own.
https://jsfiddle.net/sheraff/8g7qy0fp/8/
For example, with limits '01/01/1993' - '10/08/2018', if I begin to enter '11.08....' the max year I can proceed with is 1999.
With _min_ '01/01/2000', the max year is 2009.
To make _max_ option work correctly I have to set it to '31/12/currentYear' - enough for yearrange, but not precise max date...

**Comment by hoesein** [comment]
Hi Robin,
I also has same issue, i implement min and max value as below, so today date is max value and i can not input previous year date.
eg > today is "28-02-2020" and i need to input like "01-12-2019", now i can only type in 
"01-12-2__" and cursor is stuck in "2".

am i coding something wrong or any way to fix it's ?

Thanks for your precious plugin ...

`var today = new Date() ;
        var dd = today.getDate();
        var mm = today.getMonth() < 10 ? 0${today.getMonth()} : today.getMonth() + 1 ;
        var yyyy = today.getFullYear();
        var maxDate = ${dd}-${mm}-${yyyy} ;	
        $('.dentry_date').inputmask('datetime', {
            inputFormat : 'dd-mm-yyyy',
            placeholder : '_',
            clearIncomplete: true,
            min : '09-09-0999',
            max : maxDate
            // max : '31-12-2020'
        });`

**Comment by RobinHerbots** [comment]
@sheraff0 , @hoesein,

Just an updated fiddle for reference
https://jsfiddle.net/zo67u5hL/5/

I will have a look at the issue

**Comment by RobinHerbots** [comment]
@sheraff0, @hoesein , 

I pushed a fix in version 5.0.4-beta.31
